sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc

```diff
@@ -203,6 +203,96 @@ In the case of `ext_intel_packed` matrix memory layout, `row` and
 `col` represent the coordinates in the logical matrix before VNNI
 transformation.
 
+=== Load/Store/Fill With Out-of-Bounds Checks
```

Comment:
The check is done compared to the big matrix size comparing the address to be loaded (ptr) with the big matrix size (Height). If ptr > Height, the element should not be loaded.
if the remainder is passed, how is the implementation going to compare with it?

Reply:
With your recent change that adds `coordX` and `coordY` parameters, I see how this could work for the load and store APIs.  However, I still don't see how the fill API can work.

This API structure seems confusing to me, though, and probably less efficient.  I suspect users will get confused that the `src` pointer to `joint_matrix_load` is completely different from the `src_base` pointer to `joint_matrix_load_checked`.  It seems like it would be less confusing and more efficient to have the user pass values for `Width` and `Height` that are the *remaining* amount of memory in the big matrix.  Maybe this would be easier to explain in a meeting where we can draw pictures?

Reply:
Yes, let's discuss fill and remainder approach in a meeting.